context/v2/context_test.go

```diff
@@ -84,7 +84,7 @@ func (s *SpyResponseWriter) Write([]byte) (int, error) {
 func (s *SpyResponseWriter) WriteHeader(statusCode int) {
 	s.written = true
 }
-_
```

Comment:
Don't do this. On a v5 change why are you adding a v2 change. This is process violation. This is just a line delete and add. You should check each commits before PR

Reply:
OK